### PR TITLE
Upgrade workflow icons

### PR DESCRIPTION
**Pull request by snowystinger**
Closes 
This updates us to the latest icons. Changes listed in Test instructions.

Plan is to update @spectrum-icons/workflow by a major to signal removal of an icon and some renames.
We'll provide upgrade notes to aid in this.

One thing you'll notice is that in the chromatic changes, all DatePickers seem to have changed, this is due to moving from a 24px svg to an 18px, so the anti-aliasing is slightly different.

## ✅ Pull Request Checklist:

- [ ] Included link to corresponding [React Spectrum GitHub Issue](https://github.com/adobe/react-spectrum/issues).
- [ ] Added/updated unit tests and storybook for this change (for new code or code which already has tests).
- [ ] Filled out test instructions.
- [ ] Updated documentation (if it already exists for this component).
- [ ] Looked at the Accessibility Practices for this feature - [Aria Practices](https://www.w3.org/TR/wai-aria-practices-1.1/)

### Updated:
Actions
AEMScreens
App
AppRefresh
Circle
ColorWheel
DragHandle
Folder2Color
Gradient
Group
Labels
LinearGradient
MarginBottom
NamingOrder
NewItem
Preset
RadialGradient
RealTimeCustomerProfile
Sandbox
SelectIntersect
Ungroup

### Added:
CircleFilled
Invite
SelectSubtract
SubtractBackPath
SubtractFromSelection

### Removed:
Tableau
SelectSubstract
SubstractBackPath
SubstractFromSelection

### CODE REVIEW

**Comment by LFDanLu** [comment]
Is there a source of truth where I can look up the 2.3.2 workflow icons and compare with what we have in the chromatic stories?

**Comment by snowystinger** [comment]
> Is there a source of truth where I can look up the 2.3.2 workflow icons and compare with what we have in the chromatic stories?

@LFDanLu Not sure what you're asking for? You can always look at what's in the installed package.

We now have a couple extra icons that are duplicates of a few, purely to keep the old name of each icon around but also surface the new name.

**Comment by LFDanLu** [comment]
I was hoping for something like https://spectrum.adobe.com/page/icons/ (that one doesn't have CircleFilled I wasn't sure how up to date it was) so I could compare the appearance with what appears in chromatic but I can look at the package instead

**Comment by snowystinger** [comment]
> I was hoping for something like https://spectrum.adobe.com/page/icons/ (that one doesn't have CircleFilled I wasn't sure how up to date it was) so I could compare the appearance with what appears in chromatic but I can look at the package instead

@LFDanLu ah, I see, you'd need to look at the internal site that has them. I'll share with you directly.

File: packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+export const renameMap = {
+  A4uSelectSubtract: 'SelectSubstract',
+  A4uSubtractBackPath: 'SubstractBackPath',
+  A4uSubtractFromSelection: 'SubstractFromSelection'
```
Comment (LFDanLu) [inline packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js:5]: So just to be absolutely clear, there is an icon called `SelectSubtract` and an icon called `A4uSelectSubtract`? And thus we need to rename the A4u version to something slightly different when getting rid of the A4u from the icon name?

Just curious why we are calling it `SelectSubstract`, `substract` feels like a typo..

**Review by LFDanLu — COMMENTED** [review]
One question about substract vs subtract.

Also noted a couple of other icons were updated in look (NewItem, AEMScreens, etc), I assume these are fine even though they aren't in the PR list? Verified their appearance against that a4u URL you sent me

File: packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js (hunk 1 above)
Comment (snowystinger) [reply packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js:5]: substract was a typo, technically there only exists the new `A4uSelectSubtract` now to correct the typo, however, that's breaking, so we have this updatedNamesMap, which will map between the A4u name and a new name I want to give it, in this case, it's the old name that we want to continue to support, and I can just go directly to the name we want, don't need to worry about the prefix.

It looks like I missed those two in the description. They do look correct. Were there others?

Comment (LFDanLu) [reply packages/@spectrum-icons/workflow/scripts/updatedNamesMap.js:5]: No those were the only two actually, I was mistaken on a third. Thanks for the typo explaination, lgtm then

**Review by LFDanLu — DISMISSED** [review]

**Review by LFDanLu — DISMISSED** [review]

File: packages/@react-spectrum/actionbar/package.json [hunk 2]
```diff
@@ -46,8 +46,7 @@
     "@react-stately/collections": "^3.3.8",
     "@react-types/actionbar": "3.0.0-alpha.5",
     "@react-types/shared": "^3.12.0",
-    "@spectrum-icons/ui": "^3.2.6",
-    "@spectrum-icons/workflow": "^3.2.5"
```
Comment (snowystinger) [inline packages/@react-spectrum/actionbar/package.json:50]: extraneous dependency i found, included because of stories I think

**Review by LFDanLu — APPROVED** [review]

**Review by dannify — APPROVED** [review]

**Pull request merged**